src/extension.ts

```diff
@@ -463,13 +464,25 @@ export async function activate(context: vscode.ExtensionContext) {
 					throw new Error('Nothing to commit');
 				}
 
-				const treeItem = await treeDataProvider.getTreeItem(
-					jobHash as ElementHash,
-				);
+				const theCase = caseManager.getCase(caseHash as CaseHash);
```

Comment:
You case use `kase` in that __case__ (pun intended) since `case` is a reserved keyword.
Bonus points for that as well since __Käse__ means cheese in German :D